eureka/S3_data_reduction/plots_s3.py

```diff
@@ -116,15 +116,15 @@ def optimal_spectrum(data, meta, n):
     plt.suptitle(f'1D Spectrum - Integration {intstart + n}')
     plt.semilogy(range(subnx), stdspec[n], '-', color='C1', label='Standard Spec')
     # plt.errorbar(range(subnx), stdspec[n], yerr=np.sqrt(stdvar[n]), fmt='-', color='C1', ecolor='C0', label='Std Spec')
-    plt.errorbar(range(subnx), optspec[n], opterr[n], fmt='-', color='C2', ecolor='C2', label='Optimal Spec')
+    # plt.errorbar(range(subnx), optspec[n], opterr[n], fmt='-', color='C2', ecolor='C2', label='Optimal Spec')
```

Comment:
Why remove optimal spectrum from figure?

Reply:
oops, I'll uncomment that. This was to get around a weird bug where some values are treated as strings. There's a different issue open related to this.